src/main/java/team/serenity/model/util/SampleDataUtil.java

```diff
@@ -69,19 +77,11 @@ public static ReadOnlySerenity getSampleSerenity() {
             .collect(Collectors.toSet());
     }
 
-    /**
-     * Returns a lesson set containing the list of strings given.
-     */
-    public static Set<Lesson> getLessonSet(Lesson... lessons) {
-        return Arrays.stream(lessons)
-            .collect(Collectors.toSet());
-    }
-
     public static Question[] getSampleQuestion() {
         return new Question[]{
-            new Question(new GroupName("G01"), new LessonName("1-1"),
+            new Question(new GroupName("G01"), LESSON_1_1,
```

Comment:
Since `LESSSON_1_1` is defined, perhaps define a `GROUP_G01` as well instead of `new GroupName('G01')` ?

Reply:
Good suggestion! Ill add a new field for `GroupName` as well to be consistent :)